0-preinstall.sh

```diff
@@ -81,23 +85,32 @@ mountallsubvol () {
 }
 
 if [[ "${DISK}" =~ "nvme" ]]; then
+    partition1=${DISK}p1
     partition2=${DISK}p2
     partition3=${DISK}p3
 else
+    partition1=${DISK}1
     partition2=${DISK}2
     partition3=${DISK}3
 fi
 
+# The filesystem type for the boot partition shouldn't really matter, however
+# I didn't test such case in my installer script so I left the lines commented out
+mkfs.ext2 -L "BOOT" ${partition1}
```

Comment:
I dont think we should just ignore the user's file system preference. BTRFS users would want the snapshot capabilities..

Reply:
Ah ok! I wasn't really sure if choosing something other than ext2 would affect how the partition mounts. But I did hear it shouldn't really matter since GRUB supports those partitions.